src/Bravo3/ImageManager/Services/ImageManager.php

```diff
@@ -170,6 +186,38 @@ public function pull(Image $image)
     }
 
     /**
+     * Get meta information about the source image from the cache layer.
+     *
+     * @param Image $image
+     *
+     * @return ImageMetadata
+     */
+    public function getMetadata(Image $image)
+    {
+        $img_key = $image->getKey();
```

Comment:
Fixed. It is just I don't like to not put code within if statements if not required, for me its harder to read. But fixed.